### ISSUE TITLE
VB async call in catch and implicit interfaces still missing in VS Preview 5

### ISSUE THREAD

**Issue by vbcodec** [dotnet/roslyn#14338]
Do not forget adding especially calling async function in Catch segment of Try..Catch statement. Now I must create unpleasant workaround by copying exception to variable, test it outside Try..Catch, and call async function.

**Comment by DualBrain** [comment]
I agree.  It would be extremely helpful to be able to reuse the same methods from within the Catch block instead of me having to create an essentially duplicate code non-async method.

Add to that the fact that the feasibility of this is "very possible" since, I believe, C# does have support for this. ;-)
